jupyter/jsl_installation_script_ubuntu.sh

```diff
@@ -220,8 +220,7 @@ then
     if ! [ "$myjsl_access_token" == false ]
     then
         echo "Fetching license from MYJSL ..."
-        fetch_license_from_myjsl
-        export_json "$fetch_license_path" true
+        export_json "$fetch_license_path"
```

Comment:
Hmm... Where is the call of the `fetch_license_from_myjsl` function?

Reply:
When I ran it, I noticed that the script asks to pick the License **twice**. Once from line [#L173](https://github.com/JohnSnowLabs/spark-nlp-workshop/blob/6cf51e4ad55ce5261f9b8a7a2835b8676f8f11fb/jupyter/jsl_installation_script_ubuntu.sh#L173). This already copies the select licensed to the folder. We can just use the already saved license for the exporting it to the env variables.

Reply:
If we fetch nothing, we should display another message.

Reply:
If the user run the script without `-i` flag and only the `-r` flag, the user is choosing to use his previous installed configuration to run the environment. Should we be updating the license in that case ?

Reply:
Changed message to Reading License..